kolibri/utils/cli.py

```diff
@@ -235,9 +235,24 @@ def start(port=None, daemon=True):
 
     if not daemon:
         logger.info("Running 'kolibri start' in foreground...")
+
     else:
         logger.info("Running 'kolibri start' as daemon (system service)")
 
+    __, urls = server.get_urls(listen_port=port)
+    if not urls:
+        logger.error(
+            "Could not detect an IP address that Kolibri binds to, but try "
+            "opening up the following addresses:\n")
+        urls = [
+            "http://{}:{}".format(ip, port) for ip in ("0.0.0.0", "127.0.0.1")
```

Comment:
Per discussion in the dev meeting, we might want to remove any references to `0.0.0.0`. Also, may be worth emphasizing a distinction between local and network IPs for less IP savvy users?